package.json

```diff
@@ -76,7 +78,8 @@
         "node-fetch": "^2.6.0",
         "svgo": "^1.3.2",
         "url-loader": "^4.1.0",
-        "uxf-webpack": "^0.5.5"
+        "uxf-webpack": "^0.5.5",

```

Comment:
let's remove this and add all the stuff

Reply:
vue, babel, webpack, etc

Reply:
Addressed in https://github.com/ripe-tech/ripe-sdk-components-vue/pull/22